fintech-examples/fintech-ui/src/app/dashboard/home/home.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="row mt-5">
+<div class="row mt-5" *ngIf="cardList2 && cardList2.length">
```

Comment:
`ngIf="cardList2` shall be enough, it's javascript

Reply:
this was done by sascha. It came in due to rebase.